features/glvd/file.include/etc/update-motd.d/99-glvd

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ -x /usr/bin/glvd ]; then
+    exec /usr/bin/glvd executive-summary
```

Comment:
comment from a private chat:

this might be slow and thus undesired

should have an aggressive timeout to better not show anything than delay operations for multiple seconds

could make use of caching